Source/WebCore/css/CSSPrimitiveValue.cpp

```diff
@@ -135,6 +136,7 @@ static inline bool isValidCSSUnitTypeForDoubleConversion(CSSUnitType unitType)
     case CSSUnitType::CSS_STRING:
     case CSSUnitType::CSS_UNICODE_RANGE:
     case CSSUnitType::CSS_UNKNOWN:
+    case CSSUnitType::CSS_UNRESOLVED_COLOR:
```

Comment:
This "unresolved color" concept seems a bit odd in the CSS Value part of the codebase. `color-mix`, `currentcolor`, ... are all actual CSS values that we can find in a CSS file, but not "unresolved color". It seems that this is a rendering concept which should only exist later in the pipeline (in RenderStyle).

Reply:
The unresolved color concept is needed to support correct serialization of calc() in the specified value serialization. I would be happy to debate the name, but this seemed to have the cleanest split for me.